recipe/meta.yaml

```
@@ -13,10 +13,13 @@ source:
   git_rev: "994f92fde4ce710a4db5557b6bc22756d5f9d2b1"
 
 build:
-  number: 0
+  number: 2
   string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
   string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
   skip: true  # [win or (linux and cuda_compiler_version in (undefined, 'None', '10.2'))]
+  # as of pytorch 1.13, conda-forge only builds for CUDA 11.2+, see
+  # https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/3491
+  skip: true  # [pytorch >= "1.13" and cuda_compiler_version in ("10.2", "11.0", "11.1")]
```

Just need to do exact matching (or: stop building for multiple pytorch versions, which shouldn't be necessary given the migration infrastructure)

```suggestion
  skip: true  # [pytorch == "1.13.0" and cuda_compiler_version in ("10.2", "11.0", "11.1")]
```